DESCRIPTION

```diff
@@ -0,0 +1,98 @@
+Package: mlr3spatiotempcv
+Title: Spatiotemporal Resampling Methods for 'mlr3'
+Version: 0.0.0.9012
+Authors@R:
+    c(person(given = "Patrick",
+             family = "Schratz",
+             role = c("aut", "cre"),
+             email = "patrick.schratz@gmail.com",
+             comment = c(ORCID = "0000-0003-0748-6624")),
+      person(given = "Marc",
+             family = "Becker",
+             role = "aut",
+             email = "marcbecker@posteo.de",
+             comment = c(ORCID = "0000-0002-8115-0400")),
+      person(given = "Jannes",
+             family = "Muenchow",
+             role = "ctb",
+             email = "jannes.muenchow@uni-jena.de",
+             comment = c(ORCID = "0000-0001-7834-4717")))
+Description: Implements spatio-temporal methods to enhance the
+    experience of mlr3.
+License: LGPL-3
+URL: https://mlr3spatiotempcv.mlr-org.com/,
+    https://github.com/mlr-org/mlr3spatiotempcv,
+    https://mlr3book.mlr-org.com
+BugReports: https://github.com/mlr-org/mlr3spatiotempcv/issues
+Depends:
+    R (>= 3.5.0)
+Imports:
+    checkmate,
+    cli,
```

Comment:
I don't think we need cli, we have R's conditions and additionally a logger to output messages.

Reply:
I like {cli} but yeah I know we do not use it. Will change it.